src/TourManager/TourManager.Clients.Implementation/Mappers/Mapping.cs

```diff
@@ -0,0 +1,21 @@
+﻿using AutoMapper;
+using System;
+
+namespace TourManager.Clients.Implementation.Mappers
+{
+    public static class Mapping
+    {
+        private static readonly Lazy<IMapper> Lazy = new Lazy<IMapper>(() =>
+        {
+            var config = new MapperConfiguration(cfg => {
+                // This line ensures that internal properties are also mapped over.
+                cfg.ShouldMapProperty = p => p.GetMethod.IsPublic || p.GetMethod.IsAssembly;
+                cfg.AddProfile<ActivityClientProfile>();
+            });
+            var mapper = config.CreateMapper();
+            return mapper;
+        });
+
+        public static IMapper Mapper => Lazy.Value;
```

Comment:
Why not to just inject the mapper instead of having static class? The lazy will get removed in that case

Reply:
Cause of this is class library, it should be configured in web api's startup.cs , 
in Tour manager team as I know they configured automapper for assembly but it is not in master yet. so to prevent merge conflicts during merge - decided to configure it this way
Talked about this with   @HAghajanyan-ST  and I'm willing to refactor this part after merge and use injection